src/router.js

```diff
@@ -15,9 +15,9 @@ const routes = [
   { path: '/prices/add/multiple', name: 'price-add-multiple', component: () => import('./views/PriceAddMultiple.vue'), meta: { title: 'AddPrices', icon: 'mdi-tag-plus-outline', requiresAuth: true, breadcrumbs: [{title: 'AddPrices', disabled: true }] }},
   { path: '/prices/add/multiple/price-tag', name: 'price-add-multiple-price-tag', redirect: () => { return { path: '/prices/add/multiple' }}},
   { path: '/prices/add/multiple/receipt', name: 'price-add-multiple-receipt', redirect: () => { return { path: '/prices/add/multiple' }}},
+  { path: '/prices/add/validate', name: 'price-add-validate', component: () => import('./views/PriceAddValidate.vue'),  meta: { title: 'ValidatePrices', icon: 'mdi-checkbox-marked-circle-plus-outline', requiresAuth: true, breadcrumbs: [{title: 'Prices', disabled: false, to: '/prices' }, { title: 'Add', disabled: true }, {title: 'Validate', disabled: true }] }},
```

Comment:
in the page title/breadcrumbs, should we call it 'Validate prices', or simplify to 'Validate' ?
@TTalex @julia-gx